### PR TITLE
Adding comments that point to the official documentation

### PR DESCRIPTION
**Pull request by pjmuller**
In the ruby gem we also have comments indicating where to find the documentation for each API call.

Though it might be handy to have it in go to.

(other files still to do later)

### CODE REVIEW

File: item.go [hunk 1]
```diff
@@ -201,30 +201,36 @@ type ItemList struct {
...
 }
 
+// https://developers.podio.com/doc/items/get-item-22360
 func (client *Client) GetItem(itemId int64) (item *Item, err error) {
 	path := fmt.Sprintf("/item/%d?fields=files", itemId)
 	err = client.Request("GET", path, nil, nil, &item)
 	return
 }
 
+
```
Comment (andreas) [inline item.go]: One blankline too many?

**Comment by andreas** [comment]
Thanks! Tiny comment, but otherwise LGTM 👍

**Comment by pjmuller** [comment]
Removed the blank line ;).

**Comment by andreas** [comment]
I just squashed your two commits and merged manually (https://github.com/andreas/podio-go/commit/334a6d70346e93bec0c439842bae034beed0ffe5). Thanks for the contribution 😄

**Pull request closed without merging**